### ISSUE TITLE
keyboard shortcut missing for title bar context menu (Accessibility)

### ISSUE THREAD

**Issue by kaiyasa** [microsoft/WSL#5358]
Currently to reach the context menu for the bash console you must right click on the title bar.  For people using a screen reader like Narrator or NVDA, this is not an option.  The usual keyboard shortcut keys for this kind of task do not work.. shift+F10, ALT+space, etc.  This denies access 

# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Your Distribution version: [On Debian or Ubuntu run `lsb_release -r` in WSL]
Whether the issue is on WSL 2 and/or WSL 1: [run `cat /proc/version` in WSL]
```

# Steps to reproduce

#  Expected behavior

# Actual behavior

**Comment by therealkenc** [comment]
Please post terminal UX issues over at:

https://github.com/microsoft/terminal/issues
